apps/web-giddh/src/app/contact/aging-report/aging-report.component.ts

```diff
@@ -58,31 +52,25 @@ export class AgingReportComponent implements OnInit {
         ignoreBackdropClick: true
     };
     public isAdvanceSearchApplied: boolean = false;
-
-
     public agingAdvanceSearchModal: AgingAdvanceSearchModal = new AgingAdvanceSearchModal();
     public commonRequest: ContactAdvanceSearchCommonModal = new ContactAdvanceSearchCommonModal();
 
     @ViewChild('advanceSearch', {static: true}) public advanceSearch: ModalDirective;
     @ViewChild('paginationChild', {static: true}) public paginationChild: ElementViewContainerRef;
     @ViewChild('filterDropDownList', {static: true}) public filterDropDownList: BsDropdownDirective;
     @Output() public creteNewCustomerEvent: EventEmitter<boolean> = new EventEmitter();
-    private createAccountIsSuccess$: Observable<boolean>;
     private destroyed$: ReplaySubject<boolean> = new ReplaySubject(1);
 
     constructor(
         private store: Store<AppState>,
-        private _toasty: ToasterService,
-        private router: Router, private _agingReportActions: AgingReportActions,
-        private _contactService: ContactService,
+        private _agingReportActions: AgingReportActions,
```

Comment:
Avoid using underscores in names of private members

Reply:
@prateek-giddh this is existing code, i only removed unused code.